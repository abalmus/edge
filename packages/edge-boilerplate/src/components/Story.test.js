import { dirname } from "path"
import initStoryshots from "@storybook/addon-storyshots"

initStoryshots({
  configPath: dirname(require.resolve("edge-storybook"))
})
