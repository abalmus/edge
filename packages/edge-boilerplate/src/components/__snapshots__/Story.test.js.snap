// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Navigation basic demo 1`] = `
<nav>
  <ul>
    <li>
      <a
        href="/"
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/counter"
      >
        Counter
      </a>
    </li>
    <li>
      <a
        href="/localization"
      >
        Localization
      </a>
    </li>
  </ul>
</nav>
`;
